src/pages/permissions/actions/GroupSelectionActions.tsx

```diff
@@ -11,6 +11,7 @@ interface Props {
   loading?: boolean;
   disabled?: boolean;
   actionText?: string;
+  onEdit?: boolean;
```

Comment:
```suggestion
  isEdit?: boolean;
```
The on prefix implies this is a callback, better use "is" or "has" prefixes for booleans.